Composer/packages/client/src/pages/botProject/AppIdAndPassword.tsx

```diff
@@ -140,7 +148,34 @@ export const AppIdAndPassword: React.FC<AppIdAndPasswordProps> = (props) => {
           onChange={handleAppPasswordOnChange}
           onRenderLabel={onRenderLabel}
         />
+        <PrimaryButton
+          disabled={false}
+          styles={{ root: { width: '230px', marginTop: '15px' } }}
+          text={formatMessage('Add from publishing profile')}
+          onClick={() => {
+            setShowImportDialog(true);
+          }}
+        />
       </div>
+      {showImportDialog && (
```

Comment:
this and hidden are redundant, first one is enough imo

Reply:
I wish - the way the hidden state works within the fragment containing the dialog needs to reset its state otherwise the dialog will never open again.